src/modules/client/components/pages/CreateClientPage.tsx

```diff
@@ -39,7 +38,6 @@ const CreateClientPage: React.FC = () => {
         formRole={RecordFormRole.Client}
         onCompleted={onCompleted}
         FormActionProps={{ submitButtonText: 'Create Client' }}
```

Comment:
Removed to fix (I think) a visual bug with scrolling the form stepper.

Before: (https://qa-hmis.openpath.host/client/new)
![Screenshot 2025-01-08 at 11 52 41 AM](https://github.com/user-attachments/assets/a01f3a45-7e5f-4a2d-ac40-540c10e46ffc)

After:
![Screenshot 2025-01-08 at 1 38 00 PM](https://github.com/user-attachments/assets/073d0132-4c1b-46fe-bafc-29ced765e3fe)